frontend/src/modules/activity/components/activity-header.vue

```diff
@@ -98,11 +162,12 @@
           "
           placement="top"
         >
-          <i
-            class="ri-lg flex items-center text-base"
-            :class="computedPlatformIcon"
+          <img
             v-if="activity.platform !== 'apis'"
-          ></i>
+            :src="computedPlatformIcon"
+            class="w-4 h-4"
+            alt="Platform"
```

Comment:
Since we're now adding the `alt` attribute, maybe we could set it dynamically like 
```javascript
:alt="`${activity.platform} logo`"
```

Reply:
Done